apps/demos/testing/common.test.js

```diff
@@ -138,6 +138,7 @@ const SKIPPED_TESTS = {
   React: {
     Charts: [
       { demo: 'PiesWithEqualSize', themes: [THEME.material] },
+      { demo: 'CustomAnnotations', themes: [THEME.material] },
```

Comment:
Seems like only vectorMap test should be skipped, no? 
Do not make excess skips please

Reply:
Okay I see this test is unstable as well

Reply:
the error is on Charts so its necessary to include it on Charts

Reply:
The test error exists in Charts so its necessary to add the skipped test there instead of Vector Map